src/pykeen/datasets/cli.py

```diff
@@ -95,6 +117,18 @@ def analyze(dataset, force: bool, countplots: bool):
     fig.savefig(d.joinpath('relation_injectivity.svg'))
```

Comment:
(allow to) use a separate output path for figures?

Reply:
Done in 13f3e7c2